ios/Runner/GoogleService-Info.plist

```diff
@@ -2,20 +2,18 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-	<key>ANDROID_CLIENT_ID</key>
-	<string>823282339018-7fbpbs5cpdkb8eeq05j006mc0gavssl6.apps.googleusercontent.com</string>
 	<key>API_KEY</key>
-	<string>AIzaSyCIjs-8C_Fn7o8rXbfYI0KZSa5VKuxwbcA</string>
+	<string>AIzaSyCC0kY83Dkae-JerTEFc7rclmJ-T89wp2I</string>
```

Comment:
is this intended?

Reply:
oh, no. I should add this file to gitignore

Reply:
please do not add it to gitignore at this time (google it or ask me later); i was just asking if the changes to it were intended.

Reply:
I think it should probably be in gitignore because we have a run script for iOS that copies the correct GoogleService-Info.plist into this location whenever the app builds.